packages/react/src/components/DataTable/story/DataTable.stories.tsx

```
@@ -316,6 +316,187 @@ const sampleData = [...Array.from({ length: 5 })].map(() => ({
+      {/* Exclusion feedback: shown when in "select all" mode but some rows are unchecked */}
+      {isAllSelected && excludedIds.size > 0 && <p>Excluded IDs: {[...excludedIds].join(', ')}</p>}
+
+      <DataTable
+        enableRowSelection
+        enableSelectAll
+        manualPagination
+        columns={columns}
+        data={pageData}
+        // eslint-disable-next-line react/jsx-no-bind
+        getRowId={(row) => row.id}
+        onPaginationChange={handlePaginationChange}
```

ESLint

[error] JSX props should not use arrow functions

(react/jsx-no-bind)